test/jasmine/karma.conf.js

```diff
@@ -253,7 +252,7 @@ func.defaultConfig = {
     },
 
     browserify: {
-        transform: [pathToStrictD3, pathToShortcutPath],
```

Comment:
Don't we still want strict d3 included throughout plotlyjs while testing? What you have now is using the strict flavor in the d3 calls within the tests directly, but the danger's not there, it's in d3 calls within `src`, that we're exercising via the tests. I suppose because of the way strict-d3 is constructed, as long as one of the tests we're running imports it, it will apply to `src` as well. But that seems a bit fragile and roundabout.

Reply:
Good call. Addressed in https://github.com/plotly/plotly.js/pull/5400/commits/1496d3a43a8de7ffa057634b1560929586108ffc.